Notification/notification.js

```diff
@@ -0,0 +1,136 @@
+export class Notification {
+    constructor(dataUrl, targetBlock) {
+        this.dataUrl = dataUrl;
+        this.targetBlock = targetBlock;
+        this.active = 0;
+        this.disable = false;
+    }
+
+    async getData(dataUrl) {
+        let data = await fetch(dataUrl)
+        return await data.json();
+    }
+
+    async setData() {
+        let data = await this.getData(this.dataUrl);
+        this.data = data;
+    }
+
+    async render() {
+        await this.setData();
+        let radioInputs = '';
+        for (let { id } of this.data) {
+            radioInputs += `<input id='${id}' tabindex='${id + 3}' type="radio" name='notificationRadio' class='notification-form-radio_input' accesskey="${id}"></input>`;
+        }
```

Comment:
можно заменить на метод `map` у массива
```suggestion
        let radioInputs = this.data.map(({ id }) => `<input id='${id}' type="radio" name='notificationRadio' class='notification-form-radio_input' accesskey="${id}"></input>`);
```